src/tests/example.spec.js

```diff
@@ -27,4 +36,73 @@ test.describe('Saucedemo app basic tests', () => {
         await app.shoppingCart.removeCartItemById(0);
         await expect(app.shoppingCart.cartItems).not.toBeAttached();
     });
+
+    test('perform and verify sorting on the Inventory page', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app, page },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        // await app.inventory.sort('az');
+        // await app.inventory.sort('za');
+
+        // get all prices -> array of price
+        const pricesBeforeSort = await app.inventory.getAllPrices();
+
+        // sort all prices -> sorted array of prices -> expectedSortedResults
+        const expectedSortedResultHilo = pricesBeforeSort
+            .map((price) => {
+                const priceWithoutDollar = price.slice(1);
+                return Number(priceWithoutDollar);
+            })
+            .sort((a, b) => b - a)
+            .map((price) => `$${price}`);
```

Comment:
expectedSortedResultHilo and expectedSortedResultLohi can be extracted to helper function to the separate file

Reply:
solved with the help of ChatGPT